digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/lane/LaneService.scala

```diff
@@ -559,6 +716,16 @@ trait LaneOperations {
       dao.fetchLanesByIds( ids )
   }
 
+  def getPropertyValue(lane: PersistedLane, publicId: String) = {
+    val laneProperty = lane.attributes.find(_.publicId == publicId)
+      .getOrElse(LaneProperty("Error", Seq()))
+
+    if (laneProperty.values.nonEmpty)
+      laneProperty.values.head.value
+    else
+      None
+  }
+
```

Comment:
See comment for isSomePropertyDifferent, for both getPropertyValue methods

Reply:
Done